.codespell_ignore

```diff
@@ -0,0 +1 @@
+numer
```

Comment:
I don't know if I like global ignore for this - it's somewhat likely typo to be made. Maybe use:
- `n` and `d` or
- `num` and `den` or
- `numerator` and `denominator`
instead?

Reply:
Maybe you're right. I was trying to be consistent with Linux man-page where parameters are called `numer` and `denom`. Nevertheless i will change it.